Lib/test/test_types.py

```diff
@@ -1732,18 +1732,37 @@ def test_constructor(self):
         ns1 = types.SimpleNamespace()
         ns2 = types.SimpleNamespace(x=1, y=2)
         ns3 = types.SimpleNamespace(**dict(x=1, y=2))
+        ns4 = types.SimpleNamespace({'x': 1, 'y': 2}, x=4, z=3)
+        ns5 = types.SimpleNamespace([['x', 1], ['y', 2]], x=4, z=3)
+        ns6 = types.SimpleNamespace(UserDict({'x': 1, 'y': 2}), x=4, z=3)
```

Comment:
Is this desirable behaviour? I'd expected either a positional argument or kewword arguments to be legal, but not both at once.